tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php

```diff
@@ -1350,8 +1350,6 @@ public function testGetEmailAddressWidgetEditView()
 
         self::assertEquals($result, false);
 
-        self::assertEquals($noModuleResult, $result);
```

Comment:
why did you removing tests?

Reply:
$noModuleResult does not exist. It was breaking the test framework on my local computer.

Reply:
ok